core/db/migrate/20160303150239_add_vat_country_iso_to_price.rb

```diff
@@ -0,0 +1,7 @@
+class AddVatCountryIsoToPrice < ActiveRecord::Migration
+  def change
+    add_column :spree_prices, :vat_country_iso, :string, limit: 2, null: false, default: ''
+    add_index :spree_prices, :vat_country_iso
```

Comment:
We should probably have a combined unique index of some sort to ensure that there are not more than one price for the same set of things that should uniquely identify a price.